samples/drivers/display_shield/README.rst

```diff
@@ -0,0 +1,44 @@
+.. _display_shield-sample:
+
+Display Shield Sample
+#####################
+
+Overview
+********
+
+This sample will draw some basic rectangles onto the display.
+The rectangle colors and positions are chosen so that you can check the
+orientation of the LCD and correct RGB bit order. The rectangles are drawn
+in clockwise order, from top left corner: Red, Green, Blue, grey. The shade of
+grey changes from black through to white. If the grey looks too green or red
+at any point then the LCD may be endian swapped.
```

Comment:
What should you do if the LCD is "endian swapped"?  Is there a configuration or code change needed?

Reply:
As this is a generic display sample there is no clear solution to this issue and it is up to the user to figure it out. Could be a Kconfig that is not set correctly or a plain bug in the code.